src/elements/content-sidebar/MetadataSidebarRedesign.tsx

```diff
@@ -153,29 +154,44 @@ function MetadataSidebarRedesign({ api, elementId, fileId, onError, isFeatureEna
         }
     };
 
+    const isFullyLoaded = file && templates && templateInstances;
+    const visibleTemplateInstances = templateInstances.filter(templateInstance => !templateInstance.hidden);
+
+    const showLoading = status === STATUS.LOADING;
+    const showEmptyState = !showLoading && isFullyLoaded && visibleTemplateInstances.length === 0 && !editingTemplate;
+    const showEditor = !showEmptyState && editingTemplate;
+    const showList = !showEditor && visibleTemplateInstances.length > 0 && !editingTemplate;
```

Comment:
Can we simplify this definitions? eg. `showList` checks `!showEditor` and `!editingTemplate` which is already checked in `showEditor`.

Reply:
done!